Dockerfile

```diff
@@ -176,14 +171,10 @@ RUN \
   echo "**** cleanup ****" && \
   apk del --purge \
     build-dependencies && \
-  for cleanfiles in *.pyc *.pyo; \
-    do \
-    find /usr/lib/python3.*  -iname "${cleanfiles}" -exec rm -f '{}' + \
-    ; done && \
```

Comment:
This does save 40Mb (About 10%) but if we're going to do it we should probably do it everywhere we use python.